installers/gcp-marketplace/application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: '${OPERATOR_NAME}'
+  labels:
+    app.kubernetes.io/name: '${OPERATOR_NAME}'
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: '${OPERATOR_NAME}'
+  componentKinds:
+    - { group: core, kind: ConfigMap }
+    - { group: core, kind: Secret }
+    - { group: core, kind: Service }
+    - { group: apps, kind: Deployment }
+    - { group: batch, kind: Job }
+  descriptor:
+    description: Enterprise PostgreSQL-as-a-Service for Kubernetes
+    type: Crunchy PostgreSQL Operator
+    version: '${PGO_VERSION}'
+    icons:
+      - src: https://raw.githubusercontent.com/CrunchyData/crunchy-containers/master/images/crunchy_logo.png
```

Comment:
This doesn't work. It looks like we need to base64-encode an image as a `kubernetes-engine.cloud.google.com/icon` annotation.

- When the annotation exists, its content is used at _both_ 32x32 and 80x80 in the Console.
- The image at this URL is not square.

We can link to images in the `notes` section of the info panel using [CommonMark](https://spec.commonmark.org/), but the image can't be resized that way.

<img width="430" alt="Screen Shot 2019-11-19 at 3 52 04 PM" src="https://user-images.githubusercontent.com/106560/69189767-98c3a000-0ae4-11ea-8458-e2869278e4cb.png">